package.json

```diff
@@ -11,12 +11,11 @@
   "repository": "https://github.com/Shotbow/ChatBot",
   "dependencies": {
     "config": "^1.30.0",
```

Comment:
I didn't update `config` as it appears that could be a breaking change (1.x.x -> 3.x.x)